App/Redux/PreferencesRedux.ts

```diff
@@ -46,7 +55,44 @@ export const initialState: PreferencesState = {
   verboseUi: false,
   tourScreens: {
     wallet: true,
-    threads: true
+    threads: true,
+    notifications: true
+  },
+  services: {
```

Comment:
Cool, this works, thought just popped into my head that i wonder if it would be easier to deal with these as a tree instead of using `dependsOn`. So you'd have a root service (that is always enabled), it's children would be, for example, `notifications`, the the children of `notifications` would be all the different notification types, etc. Just a thought, if this is easy and scaleable, cool.

Reply:
pondered this one a bit more and realized, those nestings are primarily UI related, I'm going to remove them from the persisted state entirely

Reply:
done